docs/source/reference/samplers/index.rst

```diff
@@ -11,39 +11,39 @@ The :mod:`~optuna.samplers` module defines a base class for parameter sampling a
-+----------------------------------+---------------+---------------+----------------------+----------------+----------------------------------------+---------------+----------------+------------------------------+
-| Time complexity (per trial) (*)  |  :math:`O(d)` | :math:`O(dn)` | :math:`O(dn \log n)` | :math:`O(d^3)` |       :math:`O(mp^2)` (\*\*\*)         | :math:`O(dn)` | :math:`O(n^3)` |        :math:`O(d)`          |
-+----------------------------------+---------------+---------------+----------------------+----------------+----------------------------------------+---------------+----------------+------------------------------+
-| Recommended budgets (#trials)    | as many as    | number of     |      100 – 1000      |  1000 – 10000  |                100 – 10000             | as many as    |    10 – 100    |    number of combinations    |
-| (**)                             | one likes     | combinations  |                      |                |                                        | one likes     |                |                              |
-+----------------------------------+---------------+---------------+----------------------+----------------+----------------------------------------+---------------+----------------+------------------------------+
++----------------------------------+-------------------------------+-------------------------------+-------------------------------+-------------------------------+-----------------------------------------------------------------------------+-------------------------------+-------------------------------+-------------------------------------------------------------------------------+
+|                                  |         RandomSampler         |          GridSampler          |          TPESampler           |         CmaEsSampler          |                                NSGAIISampler                                |          QMCSampler           |        BoTorchSampler         |                               BruteForceSampler                               |
++==================================+===============================+===============================+===============================+===============================+=============================================================================+===============================+===============================+===============================================================================+
+| Float parameters                 |:math:`\color{green}\checkmark`|:math:`\color{green}\checkmark`|:math:`\color{green}\checkmark`|:math:`\color{green}\checkmark`|                           :math:`\blacktriangle`                            |:math:`\color{green}\checkmark`|:math:`\color{green}\checkmark`|:math:`\color{green}\checkmark` (:math:`\color{red}\times` for infinite domain)|
```

Comment:
[Note] This PR uses [`color`](https://docs.mathjax.org/en/latest/input/tex/extensions/color.html#color) of TeX extension, [`checkmark`](https://docs.mathjax.org/en/v2.7-latest/tex.html#c), [`blacktriangle`](https://docs.mathjax.org/en/v2.7-latest/tex.html#b), and [`times`](https://docs.mathjax.org/en/v2.7-latest/tex.html#t) of mathjax